content/module-reference/utility-modules/shared/metadata-editor.md

```diff
@@ -18,17 +18,13 @@ metadata entry fields
 apply
 : Apply new settings from the metadata entry fields to the selected images.
 
-reset button
-: Delete existing metadata from the selected images.
```

Comment:
does the reset button do nothing now?

Reply:
The reset button has disappeared, .... before 3.6 I think ... :)

Reply:
I was talking about the reset button at the top of the module (which I think is what the manual is referring to). It doesn't appear to do anything.

Reply:
If I'm not mistaken the reset of the module has never been wired (no default value here).
In the doc of 3.4 the button was named `clear`, next to `apply`. 
I don't know where this `reset` come from...